pkg/cmd/run.go

```diff
@@ -81,6 +81,12 @@ func RunMicroshift(cfg *config.MicroshiftConfig, flags *pflag.FlagSet) error {
 
 	os.MkdirAll(microshiftDataDir, 0700)
 
+	// Validate NodeName in config file, node-name should not be changed for an already
+	// initialized MicroShift instance (see https://issues.redhat.com/browse/USHIFT-573)
```

Comment:
I'm not sure if we want to have jira urls in code (was once told to remove them) - we can check the git blame, get PR's name and we'll have jira ticket

Reply:
Ack.